app/api/probe_dock/reports_api.rb

```diff
@@ -116,6 +122,31 @@ def serialization_options reports
             paginated_rel = paginated_rel.where('categories.name in (?)', params[:categoryNames].collect(&:to_s).to_a)
           end
 
+          if array_param?(:status) || array_param?(:kind)
+            where_clauses = []
+
+            if array_param?(:status)
+              where_clauses << 'test_payloads.passed_results_count > 0' if params[:status].include?('passed')
```

Comment:
This condition should be `test_payloads.passed_results_count > test_payloads.inactive_passed_results_count` because `passed_results_count` includes inactive results.